server/src/main/java/com/hedera/block/server/persistence/storage/write/BlockAsDirWriter.java

```diff
@@ -139,7 +139,8 @@ protected void write(@NonNull final Path blockItemFilePath, @NonNull final Block
             throws IOException {
         try (@NonNull
                 final FileOutputStream fos = new FileOutputStream(blockItemFilePath.toString())) {
-            blockItem.writeTo(fos);
+
+            BlockItem.PROTOBUF.toBytes(blockItem).writeTo(fos);
```

Comment:
hmmm, We'll have a lot of writes like this, eventually.
Might be worth creating a `writeMessage` method that's overloaded for each PBJ type we need to write to disk (also, we might want to look at using NIO channels instead of Java2 streams, probably a separate task/PR for that).

Reply:
This is good context to keep in mind for the next phase.